docs/index.md

```diff
@@ -19,7 +18,6 @@ large quantity of data and can scale with the hardware we use to analyze and sto
 
 For details and background on the project please read through the [overview](./overview.md) page.
 
-Check out the [[getting started]] page for an introduction to using Atlas in the cloud
-environment. Once you've explored the example, check out the stack language references to see
-the various types of information you can access.
-
+Check out the [getting started](./getting-started.md) page for an introduction to using Atlas in
```

Comment:
I couldn't find any info about the line width we wish to stick to here. From a cursory glance it seems like we don't go above 100 characters per line, so that's what I went for.

Reply:
Correct, we try to stay at 100 characters per line.